src/misclibs.F

```diff
@@ -4001,47 +3888,24 @@ subroutine calcmass(nstat,rstat,ruh,rvh,rmh,rho)
  
       integer i,j,k
       double precision :: tmass,var
-      double precision, dimension(nk) :: foo
       double precision :: foot
 
-      !$acc data create(foo)
 
+      tmass=0.0d0 
       !$omp parallel do default(shared) private(i,j,k)
-#ifdef _B4B06R
-      !$acc update host(rho,ruh,rvh,rmh)
-#else
-      !$acc parallel default(present) private(i,j,k) reduction(+:foot)
+      !$acc parallel default(present) private(i,j,k) reduction(+:foot,tmass)
       !$acc loop gang
```

Comment:
Do we need to add `reduction(+:tmass)` here?

Reply:
Yes, good catch.